runtime/parachains/src/disputes.rs

```diff
@@ -609,7 +697,8 @@ impl StatementSetFilter {
 				if statement_set.statements.is_empty() {
 					None
 				} else {
-					Some(statement_set)
+					// we just checked correctness when filtering.
+					Some(CheckedDisputeStatementSet::from_unchecked(statement_set))
```

Comment:
hmmm ... where exactly?

Reply:
Where the filter is constructed, in https://github.com/paritytech/polkadot/blob/de62962f7f4e21af8e5d95b7e13fbbe61ec81cf9/runtime/parachains/src/disputes.rs#L911